### ISSUE TITLE
Episode status prevents playing

### ISSUE THREAD

**Issue by Iamnome** [sentriz/gonic#432]
I am running Gonic on Docker using sentriz/gonic:latest.

I have an issue where Gonic seems to have downloaded all the podcasts from my 35 feeds, but if I check in Soundwaves there are still quite a few episodes  across different feeds showing as downloading not completed.

The number of files in the podcast directories, determined by running tree in the base podcast directory, hasn't changed in the last day or so.

**Comment by sentriz** [comment]
it's quite hard to debug this without any more info 

could visit 

http://your-gonic/rest/getPodcasts?v=v&u=USERNAME&p=PASSWORD&c=c

and find episodes where the status is not "completed", and also show the state of the filesystem for mismatches?

**Comment by Iamnome** [comment]
`<episode id="pe-11" streamId="pe-11" channelId="pd-1" title="Margaret Thatcher" description="Margaret Thatcher was one of the most controversial figures of the 20th Century. Praised by some as a fearless leader and feminist icon, and despised by others as a heartless politician, the Iron Lady caused divisions wherever she went. But why did Britain’s first female Prime Minister inspire such extreme hatred? What policies did she implement, how did they affect the communities of Britain? And what, if anything, is her legacy today? This is a Short History of Margaret Thatcher. Written by Nicole Edmunds. With thanks to Chris Collins, historian, and editor of Margaretthatcher.org. Learn more about your ad choices. Visit podcastchoices.com/adchoices" publishDate="2023-10-08T23:01:00Z" status="downloading" parent="" isDir="false" year="2023" genre="Podcast" coverArt="pd-1" size="0" contentType="audio/mpeg" suffix="mp3" duration="3235" bitRate="0" path=""/>`

I've picked this one out for an example.  it shows downloading as it's status but nothing has been downloaded for over a day.  This one shows 0 size and 0 bitrate, another one I have looked at from the Redhanded feed shows a size but 0 bitrate. This list doesn't show file names so it's very difficult to compare this to the actual files which have random names.

Is there anyway for force Gonic to restart these downloads?
